scripts/mkcloud

```diff
@@ -65,6 +65,8 @@
 #                on the mkcloud host to use instead of the one from the
 #                packages.  This will be scp'd to the admin node before use.
 #
+# qa_crowbarsetup (default /path/to/gitclone) set an optional path to qa_crowbarsetup.sh
```

Comment:
default is not /path/to/gitclone (it might) but it really looks in the same directory where mkcloud (the one that is executed currently) is located, this can be != any gitclone